.travis.yml

```diff
@@ -32,3 +37,17 @@ jobs:
       script:
         - "ci/trigger_fullstack-sdk-compat.sh"
       after_success: travis_terminate 0
+    - stage: 'NetStandard16'
+      language: csharp
+      # dotnet only works on trusty https://github.com/travis-ci/travis-ci/issues/5189
+      dist: trusty
+      mono: none
```

Comment:
if netstandard16 runs in its own stage i can disable running mono which speeds up build